schemas/bookbrainz.sql

```diff
@@ -610,6 +610,77 @@ CREATE TABLE bookbrainz._editor_entity_visits (
 ALTER TABLE bookbrainz._editor_entity_visits ADD FOREIGN KEY (editor_id) REFERENCES bookbrainz.editor (id);
 ALTER TABLE bookbrainz._editor_entity_visits ADD FOREIGN KEY (bbid) REFERENCES bookbrainz.entity (bbid);
 
+CREATE TABLE IF NOT EXISTS bookbrainz.import (
+    id SERIAL PRIMARY KEY,
+    type bookbrainz.entity_type NOT NULL
+);
+
+CREATE TABLE IF NOT EXISTS bookbrainz.creator_import (
+	import_id INT PRIMARY KEY,
+	data_id INT
+);
+ALTER TABLE bookbrainz.creator_import ADD FOREIGN KEY (import_id) REFERENCES bookbrainz.import (id);
+ALTER TABLE bookbrainz.creator_import ADD FOREIGN KEY (data_id) REFERENCES bookbrainz.creator_data (id);
+
+CREATE TABLE IF NOT EXISTS bookbrainz.edition_import (
+	import_id INT PRIMARY KEY,
+	data_id INT
+);
+ALTER TABLE bookbrainz.edition_import ADD FOREIGN KEY (import_id) REFERENCES bookbrainz.import (id);
+ALTER TABLE bookbrainz.edition_import ADD FOREIGN KEY (data_id) REFERENCES bookbrainz.edition_data (id);
+
+CREATE TABLE IF NOT EXISTS bookbrainz.publication_import (
+	import_id INT PRIMARY KEY,
+	data_id INT
+);
+ALTER TABLE bookbrainz.publication_import ADD FOREIGN KEY (import_id) REFERENCES bookbrainz.import (id);
+ALTER TABLE bookbrainz.publication_import ADD FOREIGN KEY (data_id) REFERENCES bookbrainz.publication_data (id);
+
+CREATE TABLE IF NOT EXISTS bookbrainz.publisher_import (
+	import_id INT PRIMARY KEY,
+	data_id INT
+);
+ALTER TABLE bookbrainz.publisher_import ADD FOREIGN KEY (import_id) REFERENCES bookbrainz.import (id);
+ALTER TABLE bookbrainz.publisher_import ADD FOREIGN KEY (data_id) REFERENCES bookbrainz.publisher_data (id);
+
+CREATE TABLE IF NOT EXISTS bookbrainz.work_import (
+	import_id INT PRIMARY KEY,
+	data_id INT
+);
+ALTER TABLE bookbrainz.work_import ADD FOREIGN KEY (import_id) REFERENCES bookbrainz.import (id);
+ALTER TABLE bookbrainz.work_import ADD FOREIGN KEY (data_id) REFERENCES bookbrainz.work_data (id);
+
+CREATE TABLE IF NOT EXISTS bookbrainz.discard_votes (
+    id SERIAL PRIMARY KEY,
```

Comment:
Probably better to drop "id" and use import_id/editor_id as a composite primary key here. You won't want duplicates of `(import_id, editor_id)` pairs in the database, so it should be unique, and should be enough to uniquely identify the row in the table. See for example the `bookbrainz.alias_set__alias` table.